src/components/message/message-card/MessageCardViewTypeSwitcher.tsx

```diff
@@ -0,0 +1,56 @@
+/** ****************************************************************************
+ * Copyright 2020-2020 Exactpro (Exactpro Systems Limited)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ ***************************************************************************** */
+
+import * as React from 'react';
+import MessageBody from '../../../models/MessageBody';
+import ErrorBoundary from '../../util/ErrorBoundary';
+import MessageBodyCard, { MessageBodyCardFallback } from './MessageBodyCard';
+import MessageRaw from './raw/MessageRaw';
+
+export type MessageCardViewTypeSwitcherProps = {
+	showRaw: boolean;
+	messageId: string;
+	rawContent: string | null;
+	isBeautified: boolean;
```

Comment:
this is not a switcher, but a renderer
and it should return component using switch or if else block using `MessageViewType`